bpf/lib/nodeport.h

```diff
@@ -1046,7 +1020,7 @@ static __always_inline int rev_nodeport_lb6(struct __ctx_buff *ctx, __u32 *ifind
 
 		if (!revalidate_data(ctx, &data, &data_end, &ip6))
 			return DROP_INVALID;
-
+skip_rev_dnat:
```

Comment:
The revSNAT stage will already [set the NAT-done flag](https://github.com/cilium/cilium/blob/19f2ed152a721520a286bac1ea5f62751f6b513a/bpf/lib/nodeport.h#L703), so you could move this goto-label a bit further down. (do we care about `TUNNEL_MODE` for this use-case?)

Reply:
Ack, will do, we don't care about tunnel etc in this case.